### PR TITLE
2023.3.x feature open compose record in modal

### PR DESCRIPTION
**Pull request by yonkov**

### CODE REVIEW

**Review by Fajfa — APPROVED** [review]
Almost

File: src/compose/types/page-block/record-list.ts [hunk 1]
```diff
@@ -129,7 +135,6 @@ export class PageBlockRecordList extends PageBlock {
       'hideRecordPermissionsButton',
       'editable',
       'draggable',
-      'openInNewTab',
```
Comment (Fajfa) [inline src/compose/types/page-block/record-list.ts:132]: You need to remove openInNewTab in other places aswell

Comment (Fajfa) [inline src/compose/types/page-block/record-list.ts:132]: You need to remove openInNewTab in other places aswell!

**Review by Fajfa — CHANGES_REQUESTED** [review]

File: src/compose/types/page-block/record-list.ts (hunk 1 above)
Comment (yonkov) [reply src/compose/types/page-block/record-list.ts:132]: I tried but it is breaking the backwards compatibility with the old checkbox (it is not respected anymore). I ran into two issues trying to remove the `openInNewTab` prop:
1. In it if check `if (o.openInNewTab)` i got TS error: "semantic error TS2339: Property 'openInNewTab' does not exist on type 'Partial<Options>"
2. In the same if check, if I disable ts checks, I always get undefined (value is not respected).

Comment (Fajfa) [reply src/compose/types/page-block/record-list.ts:132]: You're right, keep it like this but add a comment above so we know that it is legacy

**Comment by yonkov** [comment]
Closed due to Monorepo Migration

**Pull request closed without merging**